src/guides/v2.3/extension-dev-guide/framework/math-random.md

```diff
@@ -0,0 +1,46 @@
+---
+group: php-developer-guide
+title: Math Random
+contributor_name: Adarsh Manickam
+contributor_link: https://github.com/drpayyne
+---
+
+## Overview
+
+The [`Magento\Framework\Math\Random`]({{ site.mage2bloburl }}/{{ page.guide_version }}/lib/internal/Magento/Framework/Math/Random.php){:target="\_blank"} library provides the ability to generate random values and hashes.
+
+## Usage
+
+| Method            | Description                                                                                                                                                                          |
+| ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `getRandomString` | Generates a random string with a given string length and an optional character pool. Without the optional character pool, it uses both cases of the English alphabet and all digits. |
+| `getRandomNumber` | Generates a random number with an optional range. The range defaults to 0 as minimum and the result of the PHP function `mt_getrandmax` as the maximum.                              |
+| `getUniqueHash`   | Generates a unique hash string with an optional prefix.                                                                                                                              |
+
+## Examples
+
+For the below examples, let's assume that the `$this->mathRandom` property is an instatiated object of the `Random` class.
```

Comment:
Hi, @drpayyne !

What do you think about this option of your explanation:

```
In the examples below `$this->mathRandom` is an instatiated object of the [`Magento\Framework\Math\Random`]({{ site.mage2bloburl }}/{{ page.guide_version }}/lib/internal/Magento/Framework/Math/Random.php){:target="\_blank"} class.
```

Thank you!

Reply:
Done, thanks!